src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,33 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        if (robot.getX() - toX < 0) {
```

Comment:
```suggestion
        if (robot.getX()  < toX) {
```
we don't need compare it with 0